FusionIIIT/templates/hostelmanagement/notice_board.html

```diff
@@ -1,9 +1,11 @@
 {% load custom_tags %}
 {% block {{hall.hall_id}} %}
-    
+
     {% for notice in hall_notices|get_hall_no:hall_id %}
-        <div class="ui icon warning message">
-        <div class="ui raised segment">
+
+        <div style="margin-top: 20px;border-style: outset;border-radius: 7px;border-width: 2.5px;" 
```

Comment:
I don't think that inline CSS is a good idea here. I would like you to find the common CSS file and see if you can solve this issue from there. If not possible then we can discuss this further.

Reply:
I have removed the inline css.You can check it.